src/Map.jsx

```diff
@@ -75,7 +75,21 @@ class Map extends Component {
       { force: true },
     );
   }
-
+  getRouteBetweenStops(route) {
+    const stopSids = this.state.selectedStops.map(stop => stop.sid);
+    const routeStops = route.stops;
+    const indexArray = [];
+    for (let i = 0; i < routeStops.length; i += 1) {
```

Comment:
@EddyIonescu the point of this for loop (there's no cool built in JS method like filter or map that would work for us. forEach could work, but you can't break out of a forEach so it's less efficient)
is to get the indexes of the two selected stops so we can see if there's any points (stops)  in between them

Reply:
Seems reasonable